include/swift/ABI/MetadataValues.h

```diff
@@ -2046,8 +2052,9 @@ class TaskCreateFlags : public FlagSet<size_t> {
   explicit constexpr TaskCreateFlags(size_t bits) : FlagSet(bits) {}
   constexpr TaskCreateFlags() {}
 
-  FLAGSET_DEFINE_FIELD_ACCESSORS(Priority, Priority_width, JobPriority,
-                                 getPriority, setPriority)
+  FLAGSET_DEFINE_FIELD_ACCESSORS(CreationPriority, CreationPriority_width,
+                                 JobPriority, getCreationPriority,
+                                 setCreationPriority)
```

Comment:
These are the creation flags, so I'm not sure this was actually ambiguous in any way.  You could say "requested priority", maybe?

Reply:
Yea you're right. I think when I was grepping around for calls to `setPriority`, it hit upon both creation flags as well as jobFlags and it was a bit confusing. I will change this name to what you suggested.